src/api/v2/ResourceGroupsExtensionManager.ts

```diff
@@ -27,4 +40,37 @@ export class ResourceGroupsExtensionManager {
             }
         }
     }
+
+    async activateApplicationResourceProviders(): Promise<void> {
+        const inActiveResourceContributors =
+            vscode.extensions.all
+                .map(extension => ({ extension, resources: (extension.packageJSON as ExtensionPackage)?.contributes?.['x-azResourcesV2']?.application?.resources ?? [] }))
```

Comment:
Should make the `x-azResourcesV2` string a shared constant.

Reply:
Updated.  The compiler would have ensured proper use of the string, being part of the interface, but this *does* mean that changing it would need be done in a single place.